webservices/rest.py

```diff
@@ -329,16 +329,21 @@ def forbidden(exception):
 api.add_resource(audit.AuditCommitteeNameSearch, '/names/audit_committees/')
 
 
-def add_aggregate_resource(api, view, schedule, label):
+def add_aggregate_resource(api, id_view, schedule, label, optional_id_view=None):
```

Comment:
We can clean up this logic once we get all the work split out - this was just to test out one endpoint